draft-ietf-masque-connect-ip.md

```diff
@@ -693,8 +693,8 @@ If any of the capsule fields are malformed upon reception, the receiver of the
 capsule MUST follow the error handling procedure defined in
 {{Section 3.3 of HTTP-DGRAM}}.
 
-Upon receiving the ROUTE_ADVERTISEMENT capsule, an endpoint MAY start routing IP
-packets in these ranges to its peer.
+Upon receiving the ROUTE_ADVERTISEMENT capsule, an endpoint MAY install
+corresponding entries into its routing table.
```

Comment:
In many cases, won't this be equivalent? If I don't have a route to send on, I generally won't be able to send.

I also don't love changing the text to imply that a routing table is used by an implementation to provide access to the CONNECT-IP forwarding session. That's sometimes the case, but there are other models (like just having a flow-scoped connection). Elsewhere, we mention routing tables only as examples, not as a normative behavior.

Reply:
```suggestion
Upon receiving the ROUTE_ADVERTISEMENT capsule, an endpoint MAY update its local
state about what its peer is willing to route (subject to local policy), such as by installing
entries in a routing table.
```